docs/OnTrack/Task Submission & Redesign/Project-On-Boarding.md

```diff
@@ -0,0 +1,35 @@
+[Back to Index](Index.md)
+
+# Project On-boarding
+## Purpose of Document
+The purpose of this document is to position you, the potential contributor, such that you can contribute to the project. It is assumed that you have already begun, if not finished, the on-boarding process for the Thoth Tech company, and the OnTrack product.
+
+## Trello
+- Register a [Trello](https://trello.com/signup) account.
+- Confirm your Trello account by email (may go to Trash, so be sure to check there).
+- Join the [Trello board](https://trello.com/invite/b/elHhcRZO/289cb5bab080aeac1cf7dd9184a4fabe/task-view-taskforce)
+
+## Figma
+- Register a [Figma](https://www.figma.com/) account.
+- Confirm your Figma account by email (may go to Trash, so be sure to check there).
+- Join the [Figma project](https://www.figma.com/team_invite/redeem/SLuOCrE5wV5JuGc5tdlbBH).
+
+## What Next?
+- Become familiar with [the project epic](Epic.md)
+- Become familiar with [the user stories and features](User-Stories-and-Features.md)
+    - Are there any users that are not served in the user stories or by the features?
+- Become familiar with [the requirements](Requirements.md)
+    - Are all stakeholders sufficiently provided for, with this set of requirements?
+- Examine [Grady's proposed design](design_images/Iteration%203%20Design%201%20by%20Grady%20Ramsay.png) and ensure all requirements are met, and that the inclusion of the features are user-friendly.
```

Comment:
Remove names from formal documentation. This project will last longer than our roles, so removing names where possible would be fantastic. You could use "First proposed design" and "Second proposed design" etc.

Reply:
Thank you for the suggestion. I resolved this with [0029a50](https://github.com/thoth-tech/documentation/pull/131/commits/0029a50b924416ed774d61a6f484d136734c12b9).